csaf/src/report/mod.rs

```diff
@@ -35,7 +77,7 @@ impl Display for Title {
     }
 }
 
-struct HtmlReport<'r>(&'r ReportResult<'r>, &'r RenderOptions);
+struct HtmlReport<'r>(&'r ReportResult<'r>, &'r PathBuf, &'r Option<Url>);
```

Comment:
As we're adding too many fields, please switch to the struct form `{}`, using appropriate field names, for example:

```rust
struct HtmlReport<'r> {
  result: …,
  path: …,
  url: …,
}
```